code/modules/tutorials/newbee_tutorial.dm

```diff
@@ -0,0 +1,233 @@
+/area/tutorial/newbee
+	name = "Newbee Tutorial Zone"
+	icon_state = "green"
+	sound_group = "newbee"
+
+/obj/landmark/tutorial_clown
+	name = "The Clown You Kill To End The Tutorial"
+	icon = 'icons/map-editing/job_start.dmi'
+	icon_state = "clown"
+
+/mob/new_player/verb/play_tutorial()
+	set name = "Play Tutorial"
+	set desc = "Launch the in-game tutorial!"
+	set category = "Commands"
+
+	if (global.current_state < GAME_STATE_PLAYING)
+		boutput(usr, "You can only begin the tutorial after the game has started!")
+
+	src.client?.tutorial = new(src)
+	src.client?.tutorial.Start()
+
+/datum/tutorial_base/regional/newbee
+	name = "Newbee tutorial"
+	var/mob/living/carbon/human/tutorial/newbee = null
+	var/mob/new_player/origin_mob
+	region_type = /datum/mapPrefab/allocated/newbee_tutorial
+
+	New(mob/M)
+		..()
+		src.AddNewbeeSteps()
+		src.exit_point = pick_landmark(LANDMARK_NEW_PLAYER)
+		src.origin_mob = M
+		src.newbee = new(src.initial_turf, M.client.preferences.AH, M.client.preferences, TRUE)
+		src.owner = src.newbee
+
+	Start()
+		if (..())
+			src.origin_mob.close_spawn_windows()
+			src.origin_mob.mind.transfer_to(src.newbee)
+			src.newbee.addAbility(/datum/targetable/newbee_tutorial_exit)
+
+	Finish()
+		if(..())
+			var/mob/new_player/M = new()
+			src.newbee.mind.transfer_to(M)
+			qdel(src)
+
+/datum/tutorial_base/regional/newbee/proc/AddNewbeeSteps()
+	src.AddStep(/datum/tutorialStep/newbee/murder)
+	src.AddStep(/datum/tutorialStep/newbee/finished)
+
+/datum/tutorialStep/newbee
+	var/static/image/marker = null
+
+	New()
+		..()
+		if (!marker)
+			marker = image('icons/effects/VR.dmi', "lightning_marker")
+			marker.filters= filter(type="outline", size=1)
+
+/datum/tutorialStep/newbee/welcome
+	name = "Welcome to Space Station 13!"
+	instructions = "This tutorial will get you familiar with the basics of the game. If at any point you get stuck or want to restart the tutorial, use the 'emergency tutorial stop' verb in the <b>Commands</b> tab, top right."
```

Comment:
you probably know, but i don't see any of these when playing the tutorial
they also differ from the in-room maptext, which is probably not the best

Reply:
but yeah it's hard to do any review comments on the verbiage because it's all maptext, and i have comments

Reply:
I'd like to do more to convert this to use the tutorial steps in full, but the blob and flock tutorials use some dedicated code in their abilities to handle advancing steps. It should be possible to make the tutorial steps generate maptext instead of having it hard coded; my intention is to do that but I wanted to get something playable first.